near-contract-standards/src/fungible_token/core.rs

```diff
@@ -91,14 +91,14 @@ pub trait FungibleTokenCore {
     fn ft_transfer_call(
         &mut self,
         receiver_id: AccountId,
-        amount: U128,
+        amount: NearToken,
```

Comment:
Hold on, these are not NearTokens, so keep these U128.

Note: Most probably all the changes in FT implementation needs to be reverted.